docs/integrations/generate.js

```diff
@@ -0,0 +1,245 @@
+    category: 'social',
+    logoFilename: 'alipay.svg',
+  },
+  {
+    id: 'kakao',
+    name: 'Kakao',
+    description: 'Kakao is a famous social network service provider in South Korea.',
+    category: 'social',
+  },
+  {
+    id: 'naver',
+    name: 'Naver',
+    description: 'Naver is the most leading internet service provider in South Korea.',
+    category: 'social',
+  },
+  {
+    id: 'azuread',
+    name: 'Microsoft',
+    description: 'Microsoft Azure Active Directory is a leading AD provider.',
+    category: 'social',
+  },
+  {
+    id: 'feishu-web',
+    name: 'Feishu',
+    description: 'Feishu is an enterprise collaboration platform developed by ByteDance.',
+    category: 'social',
+    logoFilename: 'feishu.svg',
+  },
+  {
+    id: 'huggingface',
+    name: 'Hugging Face',
+    description:
+      'Hugging Face is a machine learning (ML) and data science platform and community that helps users build, deploy and train machine learning models.',
+    category: 'social',
+  },
+  {
+    id: 'wecom',
+    name: 'WeCom',
+    description:
+      'WeCom is a cross-platform instant messaging app for team. It is the enterprise version of WeChat.',
+    category: 'social',
+  },
+  {
+    id: 'oauth2',
+    name: 'OAuth 2.0 (Social)',
+    description:
+      'The OAuth 2.0 authorization framework enables a third-party application to obtain limited access to an HTTP service.',
+    category: 'social',
+    logoFilename: 'oauth.svg',
+  },
+  {
+    id: 'oidc',
+    name: 'OIDC (Social)',
+    description: 'OpenID Connect 1.0 is a simple identity layer on top of the OAuth 2.0 protocol.',
+    category: 'social',
+  },
+  {
+    id: 'aws-ses',
+    name: 'AWS Direct Mail',
+    description:
+      'Amazon SES is a cloud email service provider that can integrate into any application for bulk email sending.',
+    category: 'email',
+    logoFilename: 'aws.svg',
+  },
+  {
+    id: 'sendgrid-email',
+    name: 'SendGrid Email',
+    description: 'SendGrid is a communication platform for transactional and marketing email.',
+    category: 'email',
+    logoFilename: 'sendgrid.svg',
+  },
+  {
+    id: 'aliyun-dm',
+    name: 'Aliyun Direct Mail',
+    description: 'Aliyun provides cloud computing services to online businesses.',
+    category: 'email',
+    logoFilename: 'aliyun.svg',
+  },
+  {
+    id: 'mailgun',
+    name: 'Mailgun',
+    description:
+      'Mailgun is an email delivery service for sending, receiving, and tracking emails.',
+    category: 'email',
+  },
+  {
+    id: 'smtp',
+    name: 'SMTP',
+    description:
+      'The SMTP is an internet standard communication protocol for electronic mail transmission.',
+    category: 'email',
+  },
+  {
+    id: 'twilio-sms',
+    name: 'Twilio SMS',
+    description: 'Twilio provides programmable communication tools for phone calls and messages.',
+    category: 'sms',
+    logoFilename: 'twilio.svg',
```

Comment:
is it possible to read a file for these metadata props?

Reply:
It is possible, but let's talk about it offline to see if it is necessary.

Reply:
Discussed offline, and created a task to improve the script. https://linear.app/silverhand/issue/LOG-8950/improve-the-pull-connector-readme-script